components-java-sdk/src/main/java/io/dapr/components/domain/pubsub/PullMessagesRequest.java

```diff
@@ -21,9 +21,10 @@ public record PullMessagesRequest(Topic topic, String ackMessageId, Optional<Str
    * Conversion constructor.
    *
    * @param other The Protocol Buffer representation of a PullMessagesRequest.
+   * @return The provided protocol buffer object converted into the local domain.
    */
-  public PullMessagesRequest(dapr.proto.components.v1.Pubsub.PullMessagesRequest other) {
-    this(new Topic(other.getTopic()),
+  public static PullMessagesRequest fromProto(dapr.proto.components.v1.Pubsub.PullMessagesRequest other) {
+    return new PullMessagesRequest(new Topic(other.getTopic()),
```

Comment:
If the proto has no topic; should the POJO topic property be empty (i.e. `null`) as well?

Reply:
In proto3+java, null or empty protos end up receiving the default value which, for Strings is an empty string> but let me verify that in a test.

Reply:
Because Topic is, itself, a message and not a base type, proto3 defines a `hasXXX` method for it. I am using it for check for `topic` existence on `Topic#fromProto(Pubsub.PullMessagesRequest)`.

Because I split the parsing of `Pubsub.PullMessagesRequest` into a Topic::fromProto and `PullMessageAcknowledgement::fromProto` it might be the case that it would be better to just delete this method from `PullRequestMessage` or remove this class completely, as it is not being exposed in any interface. 

Thoughts?